tests/Bundle/ServiceInstantiationTest.php

```diff
@@ -23,15 +24,14 @@
 use Symfony\Component\Filesystem\Filesystem;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\KernelInterface;
 
 class ServiceInstantiationTest extends WebTestCase
 {
-    protected function setUp(): void
+    public static function setUpBeforeClass(): void
```

Comment:
would you mind this change? on my computer it greatly improves test performances:

- before: `Time: 01:53.571`
- after: `Time: 01:11.453`